features/FEATURE_UVISOR/includes/uvisor/api/inc/uvisor-lib.h

```diff
@@ -17,6 +17,8 @@
 #ifndef __UVISOR_API_UVISOR_LIB_H__
 #define __UVISOR_API_UVISOR_LIB_H__
 
+#include "includes/uvisor-lib/uvisor-lib.h"
```

Comment:
Looks like at least `cmsis/core_cmSecureAccess.h` needs to include `uvisor-lib/uvisor-lib.h` instead of just `uvisor-lib.h`.

Note that `uvisor-lib/uvisor-lib.h` includes `api/inc/uvisor-lib.h`, which is why we shouldn't include `uvisor-lib/uvisor-lib.h` from `api/inc/uvisor-lib.h`.